### DISCUSSION TITLE
posts/deploy-telegram-bot-as-azure-function-app

### DISCUSSION THREAD

**Discussion by giscus** [marcveens/dev-blog#32] [Announcements]
# posts/deploy-telegram-bot-as-azure-function-app

How to register and deploy a Node.js Telegram bot as an Azure Function App

https://www.marcveens.nl/posts/deploy-telegram-bot-as-azure-function-app

**Comment by anton-bocharov-3shape** [comment]
nice one, I like the trick with timer for Azure function!
